tools/mpremote/mpremote/main.py

```diff
@@ -228,6 +229,26 @@ def argparse_mip():
     return cmd_parser
 
 
+def argparse_romfs():
+    cmd_parser = argparse.ArgumentParser(description="manage ROM partitions")
+    _bool_flag(cmd_parser, "mpy", "m", True, "download as compiled .mpy files (default)")
```

Comment:
```suggestion
    _bool_flag(cmd_parser, "mpy", "m", True, "Automatically compile .py files to .mpy when buliding the ROMFS (default)")
```

(Suggested rewording as the verb "download" is unclear in this context.)

Reply:
Reworded as suggested.